docs/versioned_docs/version-v1.1.x/architecture.md

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
we don't need v1.1 docs yet, this should be done after cutting a release

Reply:
Done, thanks